src/System.Runtime/src/System.Runtime.Forwards.cs

```diff
@@ -1,6 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements. 
-// The .NET Foundation licenses this file to you under the MIT license. 
-// See the LICENSE file in the project root for more information. 
-
-[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.InteropServices.GCHandle))]
-[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.InteropServices.GCHandleType))]
```

Comment:
@ericstj This change also removes explicit type forwards of GCHandle from System.Runtime. Should I leave this change in here?